cypress/integration/specialFeatures/preRollAds/index.js

```diff
@@ -0,0 +1,38 @@
+import config from '../../../support/config/services';
+import getPaths from '../../../support/helpers/getPaths';
+import testsForCanonicalOnly from './testsForCanonicalOnly';
+// import testsForAMPOnly from './testsForAMPOnly';
+
+// eslint-disable-next-line no-unused-vars
+const hasAds = serviceName => {
+  return true;
```

Comment:
We're going to need to implement this logic - either using a hardcoded environment config like in other Cypress tests(https://github.com/bbc/simorgh/blob/latest/cypress/support/config/envs.js#L19) and/or check the toggles service.